.gitlab-ci.yml

```diff
@@ -13,7 +13,9 @@ stages:
 
 include:
   - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-check-commits-signoffs.yml'
+    file: '.gitlab-ci-check-commits.yml'
```

Comment:
This repo is not part of the release, use `check-commits-signoffs` instead of `check-commits`.